docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -63,11 +69,40 @@ $ npx express-generator
      $ DEBUG=express:* npm start
 ```
 
-Your source code has now been generated and created in the directory.
 
 </TabItem>
+  <TabItem value="dotnet" label=".NET">
+
+  For .NET we will use the [ASP.NET Core MVC](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-6.0&tabs=visual-studio-code) example. 
+
+  ASP.NET MVC is a web application framework that implements the model-view-controller (MVC) pattern.
+
+  1. Generate an example project:
+
+```console
+$ dotnet new mvc --name app
+
+Welcome to .NET 6.0!
+---------------------
+SDK Version: 6.0.104
+
+...
+
+The template "ASP.NET Core Web App (Model-View-Controller)" was created successfully.
+This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/6.0-third-party-notices for details.
+
+Processing post-creation actions...
+Running 'dotnet restore' on /Users/user/app/app.csproj...
+  Determining projects to restore...
+  Restored /Users/user/app/app.csproj (in 84 ms).
+Restore succeeded.
+```
```

Comment:
@cdrage this should have been tabbed. If you look at how it's rendered, it's not really under the point 1 that you have written above. Something like this:
```suggestion
    ```console
    $ dotnet new mvc --name app
    
    Welcome to .NET 6.0!
    ---------------------
    SDK Version: 6.0.104
    
    ...
    
    The template "ASP.NET Core Web App (Model-View-Controller)" was created successfully.
    This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/6.0-third-party-notices for details.
    
    Processing post-creation actions...
    Running 'dotnet restore' on /Users/user/app/app.csproj...
      Determining projects to restore...
      Restored /Users/user/app/app.csproj (in 84 ms).
    Restore succeeded.
    ```

```